src/main/java/nl/ordina/jobcrawler/model/VacancyURLs.java

```diff
@@ -0,0 +1,15 @@
+package nl.ordina.jobcrawler.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
```

Comment:
@Data in plaats van @Getter en @Setter
En @Builder overwegen

Reply:
Gewijzigd in https://github.com/mostafa-221/jobcrawler-backend/commit/b2d4295393dbd1fdb6937812aeb913bce41c50f1

Overigens builder toegepast in de YachtVacancyScraper om dit direct toe te passen.